models/oakland_candidate.rb

```diff
@@ -30,6 +30,29 @@ def as_json(options = nil)
       last_name: last_name,
       ballot_item: office_election.id,
       office_election: office_election.id,
+
+      # contribution data
+      supporting_contribution_data: {
```

Comment:
nitpick, I prefer to leave off the `_data`, since all of this is data and feels redundant. I think having an entity named `supporting_contributions` or `supporting_money` is totally cool and very descriptive of the domain we're working in.